fixtures/packages/polymer/expected/test/unit/shady-unscoped-style-import-css.js

```diff
@@ -1 +0,0 @@
-
```

Comment:
Note for other reviewers: This change is due to how we previously incorrectly marked this file for conversion. We now correctly recognize that this file is imported as CSS.